build/preinstall.sh.cmake

```diff
@@ -29,7 +29,7 @@ mecho()
 
 ###############################################################################
 
-COMPILEFOR=@COMPILEFOR@
+SYSTEM_NAME=@CMAKE_SYSTEM_NAME@
```

Comment:
why not name the var identically? its not a problem adding CMAKE_ here to the name, you would at least know what was meant?